lib/active_record/connection_adapters/postgis_adapter.rb

```diff
@@ -17,7 +17,6 @@
 require_relative "postgis/oid/spatial"
 require_relative "postgis/oid/date_time"
 require_relative "postgis/type" # has to be after oid/*
-require_relative "postgis/create_connection"
```

Comment:
This is the only actual needed change according to the current test suite, but I'm trying with the whole rails test suite to be sure we do not miss anything